src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -190,48 +190,28 @@ class RandomFourierGaussPreproc: public RandomMixin<DensePreprocessor<float64_t>
 	 */
 	float64_t kernelwidth;
 
-	/** dimension of input features
-	 * width of gaussian kernel in the form of exp(-x^2 / (2.0 kernelwidth^2) ) NOTE the 2.0 and the power ^2 !
-	 */
-	float64_t cur_kernelwidth;
-
-	/** desired dimension of input features as set by void set_dim_input_space(const int32_t dim)
-	 *
-	 */
-	int32_t dim_input_space;
-
-	/** actual dimension of input features as set by bool init_randomcoefficients() or void set_randomcoefficients
-	 *
-	 */
-	int32_t cur_dim_input_space;
-
-
 	/** desired dimension of output features  as set by void set_dim_feature_space(const int32_t dim)
 	 *
 	 */
 	int32_t dim_feature_space;
 
-	/** actual dimension of output features as set by bool init_randomcoefficients() or void set_randomcoefficients
-	 *
-	 */
-	int32_t cur_dim_feature_space;
-
 	/**
-	 * tests whether rf features have already been initialized
+	 * random coefficient
+	 * length = dim_feature_space
 	 */
-	bool test_rfinited() const;
+	SGVector<float64_t> random_offset;
```

Comment:
if you are changing names, apply a "m_" prefix

Reply:
Would you mind if I change dim_feature_space as dim_feats?. Because ```m_dim_feature_space``` is just too big. Similarly ```m_rand_weights``` and ```m_rand_offset```